src/client/components/PageHeaderComponent/PageHeader.styles.css

```diff
@@ -0,0 +1,13 @@
+@import url('https://fonts.googleapis.com/css2?family=Chakra+Petch:wght@400;500;600&display=swap');
+
+h1 {
+  font-family: 'Chakra Petch', sans-serif;
+  letter-spacing: 7px;
+  color: #fff;
+  margin-left: 5em;
+}
+
```

Comment:
Should we skip using h1, maybe we can create a class for the h1, so it does not affect globally? What do you say?

Reply:
Thank you Tithi, I think you are absolutely right, I will change it asap.